src/gt4py/next/program_processors/runners/dace/transformations/auto_optimize.py

```diff
@@ -137,17 +137,19 @@ def gt_auto_optimize(
                 repl=constant_symbols,
                 simplify=True,  # Simplify again after.
                 simplify_at_entry=False,
-                validate=validate,
+                validate=False,
```

Comment:
to me it feels like for debugging you might want to have the option to turn it on?